app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -35,39 +39,87 @@
   }
 }
 
+.js-enabled .gem-c-feedback__prompt-questions {
+  display: inline-block;
+}
+
+.gem-c-feedback__prompt-questions {
+  position: relative;
+  text-align: center;
+  border-bottom: 1px solid govuk-colour("white");
+  padding: govuk-spacing(5) govuk-spacing(5) govuk-spacing(3) govuk-spacing(5);
+  box-sizing: border-box;
+  width: 100%;
```

Comment:
This width and the position relative on line 47 don't appear to do anything, but I could be wrong.